vms/authentication/tests/test_login.py

```diff
@@ -37,8 +35,8 @@ def setUpClass(cls):
         super(TestAccessControl, cls).setUpClass()
 
     def setUp(self):
-        admin = create_admin()
-        volunteer = create_volunteer()
+        create_admin()
```

Comment:
```setup``` and ```teardown``` will be called, can you please reason the removed of assignment of these two to ```admin``` and ```volunteer```?